ios/keyman/Keyman/Keyman/Keyman.entitlements

```diff
@@ -2,6 +2,11 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>com.apple.developer.associated-domains</key>
+	<array>
+		<string>applinks:keyman.com</string>
+		<string>applinks:keyman-staging.com</string>
```

Comment:
Do we need to have something that restricts to a specific path here?

Reply:
Nope.  That's what https://github.com/keymanapp/keyman.com/pull/141 was for.  I think the in-app entitlements stuff says "check these sites for the apple-app-site-association file", or something like that.

Well, that and https://github.com/keymanapp/keyman/pull/3466/files#diff-42c700751fafa933a17969ebb675653fR102-R139.